.github/workflows/release-ghcr-main.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright The ORAS Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: release-ghcr-main
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Is this pretty standard as a build image?

Reply:
This is the environment of the runner in which we execute our job. In this case, it's an `ubuntu-latest` machine. 
If you want the builder instance to be specific, then we can use `runs-on: ubuntu-22.04`. 
I'm in favor of keeping it specific, but in other workflow we use the latest instance, so I preferred the same. 

https://github.com/oras-project/oras/blob/9ecf303eeb2620dfe3ddf15721999d1f7e8fd4bf/.github/workflows/release-ghcr.yml#L23